packages/browser/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@shopify/browser",
+  "version": "1.0.0",
```

Comment:
The original package in `web` didn't maintain any separate versioning of its own, but the [docs](https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100) say that code already in production should apparently be at least version `1.0.0`.